docs/sources/operators-guide/deploy-grafana-mimir/getting-started-helm-charts/_index.md

```diff
@@ -79,6 +79,11 @@ Using a custom namespace solves problems later on because you do not have to ove
          # empty, disabled.
    ```
 
+   > **Note:** You can use `helm show values grafana/mimir-distributed` to see all configurable parameters for helm install.
+   > However, refrain from copy and paste the whole thing because all parameters default value are already set and you should only
```

Comment:
The fact that there are defaults is not a problem. I would just say that the values file can change in backwards incompatible ways between major versions and you would have to merge every change from the new version to keep it working - not just what you actually customized.

Reply:
Rephrased the note.